### PR TITLE
Enable ammonite for test:console on l_space

### PR DESCRIPTION
**Pull request by parkan**
If we use Build.scala, we can use real Scala!

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -77,4 +75,61 @@ object LSpaceBuild extends Build{
...
+  import com.orientechnologies.orient.core.Orient
+  import org.apache.tinkerpop.gremlin.orientdb.OrientGraphFactory
+  import gremlin.scala._
+  import io.mediachain.Types._
+  import io.mediachain.Traversals.{GremlinScalaImplicits, VertexImplicits}
+  import io.mediachain.util.orient.MigrationHelper
+
+  Orient.instance.removeShutdownHook()
+  lazy val graph = MigrationHelper.newInMemoryGraph()
+  """.split("\n").mkString("; ")
```
Comment (parkan) [inline project/Build.scala]: this is super gross but it works!

Comment (yusefnapora) [inline project/Build.scala]: hah! awesome

**Comment by parkan** [comment]
Needs some cleanup, ended up committing some extra pieces

File: project/Build.scala (hunk 1 above)
Comment (bigs) [inline project/Build.scala]: LOL werk werk werk

**Comment by bigs** [comment]
@parkan wna clean and merge?

**Comment by parkan** [comment]
yeah, that's enqueued

**Comment by parkan** [comment]
I think this is pretty much good. I've consolidated the `build.sbt`s into `Build.scala`

**Comment by yusefnapora** [comment]
nice; I'm into it 👍

**Comment by parkan** [comment]
Merging if no one objects

**Pull request merged**